recipes/cairo/meson/conanfile.py

```diff
@@ -75,12 +67,14 @@ def export_sources(self):
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
-        if self.settings.os != "Linux":
+        if self.settings.os not in ["Linux", "FreeBSD"]:
             del self.options.with_xlib
             del self.options.with_xlib_xrender
             del self.options.with_xcb
             del self.options.with_symbol_lookup
-        if self.settings.os in ["Macos", "Windows"]:
+        if self.settings.os == "Windows" or is_apple_os(self):
+            del self.options.with_opengl
+        if Version(self.version) >= "1.17.8":
```

Comment:
Here there might be a double delete of with_opengl option, this is the reason of v2 pipeline failure.

To replace with:

```python
if Version(self.version) >= "1.17.8" or self.settings.os == "Windows" or is_apple_os(self):
    del self.options.with_opengl
```